synapse/replication/http/_base.py

```diff
@@ -104,6 +111,8 @@ class ReplicationEndpoint(metaclass=abc.ABCMeta):
     RETRY_ON_CONNECT_ERROR = True
     RETRY_ON_CONNECT_ERROR_ATTEMPTS = 5  # =63s (2^6-1)
 
+    WAIT_FOR_STREAMS = True
```

Comment:
Should this be annotated as `ClassVar[bool]`? (We always look it up via `cls.WAIT_FOR_STREAMS` AFAICS)

Reply:
Err, yeah can do!